src/app/GitCommands/Settings/AppSettings.cs

```diff
@@ -1960,33 +1966,39 @@ public static bool RepoObjectsTreeShowSubmodules
             set => SetBool("RepoObjectsTree.ShowSubmodules", value);
         }
 
+        public static int RepoObjectsTreeFavoritesIndex
+        {
+            get => GetInt("RepoObjectsTree.FavoritesIndex", 0);
+            set => SetInt("RepoObjectsTree.FavoritesIndex", value);
+        }
+
         public static int RepoObjectsTreeBranchesIndex
         {
-            get => GetInt("RepoObjectsTree.BranchesIndex", 0);
+            get => GetInt("RepoObjectsTree.BranchesIndex", 1);
```

Comment:
I think this change is safe because the name of the setting is not changed but only its default value. These settings affect just the sequence in the left panel.
I was afraid that it might fail to insert the favorites as first node but it worked.